nilearn/plotting/html_stat_map.py

```diff
@@ -75,6 +75,25 @@ def _rgb_to_papaya(x, cmap):
     return cmap
 
 
+def _view_stat_map_change_warning():
+    """
+    While view_stat_map() is undergoing development, raises a warning
+    that view_stat_map() may change in future.
+    
+    Raises
+    ------
+    FutureWarning
+    """
+    import warnings
+    warnings.simplefilter('always')
+    message = ('"{}()" is under actve development and its interface and functionality'
+               'may change in future without warning. Please do not rely on it '
+               'in critical and production environments.'
+        .format(view_stat_map.__name__)
```

Comment:
Why not hardcoding the name of the function? This adds complexity for no good reason.